content/docs/getting-started/edit-content.md

```diff
@@ -0,0 +1,129 @@
+
+## Create & Register a Form
+
+We will use the `useForm` hook to [create the form](/docs/plugins/forms#creating-forms).
+
+Simply creating the form doesn't make it render in the sidebar, we also need to [register it](/docs/plugins/forms#registering-forms) with the CMS. With the `usePlugin` hook, we'll register the form so it will be rendered in the sidebar.
+
+<!-- Todo provide explanation of why these are two steps -->
+
+> _Fun-Fact:_ Forms are a type of [plugin](/docs/plugins) in Tina.
+
+### The Steps
+
+1. Import the `useForm` and `usePlugin` hooks
+2. Define the [form configuration](/docs/plugins/forms#form-configuration)
+3. Create a form with `useForm` and the config object
+4. Register the form with `usePlugin`
+5. Render the data returned from `useForm`
+
+**src/App.js**
+
+```js
+// 1. Import `useForm` and `usePlugin`
+import { TinaProvider, TinaCMS, useCMS, useForm, usePlugin } from 'tinacms'
+
+//...
+
+const pageData = {
+  title: 'Tina is not a CMS',
+  body: 'It is a toolkit for creating a custom CMS.',
+}
+
+function PageContent() {
+  // 2. Define the form configuration object
+  const formConfig = {
+    id: 'tina-tutorial-index',
+    label: 'Edit Page',
+    fields: [
+      {
+        name: 'title',
+        label: 'Title',
+        component: 'text',
+      },
+      {
+        name: 'body',
+        label: 'Body',
+        component: 'textarea',
+      },
+    ],
+    initialValues: pageData,
+    onSubmit: async () => {
+      window.alert('Saved!')
+    },
+  }
+
+  // 3. Create the form
+  const [editableData, form] = useForm(formConfig)
+
+  // 4. Register it with the CMS
+  usePlugin(form)
+
+  return (
+    <section className="App-header">
+      <img src={logo} className="App-logo" alt="logo" />
+      {/**
+       * 5. Render the `editableData` returned from `useForm`
+       */}
+      <h1>{editableData.title}</h1>
+      <p>{editableData.body}</p>
+      <ExitButton />
+    </section>
+  )
+}
+
+//...
+```
+
+## Edit the data
+
+Head back to http://localhost:3000, enable the CMS, open the sidebar and try to update the content. You should now be able to edit the title and body copy on the demo!
+
+![tina-tutorial-edit-data](/img/getting-started/edit-data.png)
+
+## Form Configuration
+
+To gain a little clarity into how this is working, let's look closer at the `formConfig`.
+
+`useForm` needs a [form configuration object](/docs/plugins/forms#form-configuration) with properties that determine how the form behaves on load and save, what fields are available, and other metadata.
+
+While there are other form config properties, the key ones to examine in the example above are `id`, `fields`, `initialValues`, and `onSubmit`. We'll look closely at each of these now.
+
+### _id_
+
+The `id` is unique identifier for the form. For this simple example the `id` is hard-coded, however it is good practice to set `id` to as a unique identifier from the content being edited.
```

Comment:
```suggestion
The `id` is a **unique identifier** for the form. The `id` is hard-coded in our example, which is OK because the `PageContent` component is only used to render the home page. When you create a form inside of a component that is used on multiple pages, you will want to take care to ensure the form's `id` uniquely identifies the content currently being edited. Typically, you would use the content's filename, or a unique key returned from a content API.
```

I want to be a little stronger than just saying "good practice", as certain things will not work right if IDs aren't unique